packages/lwc-language-server/src/javascript/compiler.ts

```diff
@@ -160,6 +160,7 @@ export function toVSCodeRange(babelRange: SourceLocation): Range {
 }
 
 export function extractAttributes(metadata: Metadata, uri: string): { privateAttributes: AttributeInfo[]; publicAttributes: AttributeInfo[] } {
+    console.log('I hit here');
```

Comment:
I know this is already merged, but this `console.log` should probably be removed.